Index: lab_1/task_2/key.json
===================================================================
diff --git a/lab_1/task_2/key.json b/lab_1/task_2/key.json
deleted file mode 100644
--- a/lab_1/task_2/key.json	(revision 1b4bad32b7d100fe7ed0ea60f9fe53eac6cd1005)
+++ /dev/null	(revision 1b4bad32b7d100fe7ed0ea60f9fe53eac6cd1005)
@@ -1,38 +0,0 @@
-{
-  "Я": " ",
-  "Д": "Е",
-  " ": "А",
-  "4": "О",
-  "t": "И",
-  "М": "Н",
-  "Р": "Т",
-  "О": "Р",
-  "П": "С",
-  "Б": "В",
-  "Л": "М",
-  "К": "Л",
-  "5": "П",
-  "2": "Д",
-  "Ь": "Я",
-  "Щ": "Ы",
-  "У": "Х",
-  "Ъ": "Ь",
-  "Й": "К",
-  "r": "З",
-  "Х": "Ч",
-  ">": "У",
-  "Т": "Ф",
-  "Е": "Ж",
-  "1": "Г",
-  "Ы": "Ю",
-  "А": "Б",
-  "И": "Й",
-  "Ц": "Ш",
-  "?": "Ъ",
-  "Ф": "Ц",
-  "Ч": "Щ",
-  "w": "Э"
-}
-
-
-
Index: lab_1/main.py
===================================================================
diff --git a/lab_1/main.py b/lab_1/main.py
deleted file mode 100644
--- a/lab_1/main.py	(revision 1b4bad32b7d100fe7ed0ea60f9fe53eac6cd1005)
+++ /dev/null	(revision 1b4bad32b7d100fe7ed0ea60f9fe53eac6cd1005)
@@ -1,180 +0,0 @@
-"""
-это самое
-"""
-import json
-
-RU_ALPH = 'АБВГДЕЖЗИЙКЛМНОПРСТУФХЦЧШЩЪЫЬЭЮЯ'
-
-
-def read_file(file_name: str) -> str:
-    """
-    reading file
-
-    :param file_name: path to file
-    :return:  string in one line (withoun \n)
-    """
-    try:
-        return ''.join(open(file_name, 'r').readlines()) # .replace('\n', '   ')
-    except Exception as e:
-        return 'reading error:' + str(e)
-
-
-def write_file(file_name: str, data: str) -> None:
-    """
-    writing to file
-
-    :param file_name: path to file
-    :param data: data for writing
-    :return None:
-    """
-    try:
-        open(file_name, 'w').write(data) # .replace('   ', '\n')
-    except Exception as e:
-        print(e)
-
-
-def rot16(string: str) -> str:
-    return ''.join(map(lambda c: chr(ord(c) + 16) if c.isalpha() and c.lower() <= 'п' else
-                                 chr(ord(c) - 16) if c.isalpha() else c, string))
-
-
-def task_1(file_to_read: str, file_to_write: str) -> None:
-    """
-    writing encrypted text to file using function rot16
-
-    :param file_to_read:
-    :param file_to_write:
-    :return:
-    """
-    write_file(file_to_write, rot16(read_file(file_to_read)))
-
-
-def global_symbols_frequency() -> dict:
-    """
-    count stat for symbols in global using
-
-    :return:
-    """
-    frequency_lst = [i.replace('§', ' ') for i in read_file('task_2/frequency_symbols_based.txt').split('\n')]
-    dictionary = dict()
-    symbols, frequency = \
-        [frequency_lst[i][0] for i in range(len(frequency_lst))], \
-        [float(frequency_lst[i][4:12]) for i in range(len(frequency_lst))]
-
-    for ind, key in enumerate(symbols):
-        dictionary[key] = frequency[ind]
-    return dictionary
-
-
-def encrypted_text_symbols_frequency() -> dict:
-    """
-    complete dictionary with symbols frequency in the text
-
-    :return: text's symbols dictionary
-    """
-    encrypted_text = read_file('task_2/cod8.txt')
-    symbols_dict = dict()
-    for key in set(encrypted_text):
-        symbols_dict[key] = float(round(encrypted_text.count(key) / len(encrypted_text), 6))
-    return symbols_dict
-
-
-# def get_equals_dict() -> None:
-#     """
-#     function used for keys permutations
-#
-#     :return:
-#     """
-#     gsf = global_symbols_frequency()
-#     etsf = encrypted_text_symbols_frequency()
-#
-#     s_encrypted_text_frequency = sorted(etsf.values(), reverse=True)
-#
-#     sorted_text_dict = dict()
-#     for frequency in s_encrypted_text_frequency:
-#         for etsf_key in etsf.keys():
-#             if etsf[etsf_key] == frequency:
-#                 sorted_text_dict[etsf_key] = frequency
-#
-#     cor_key = [' ', 'Е', 'А', 'О', 'И', 'Н', 'Т', 'Р', 'С', 'В', 'М', 'П', 'Л', 'Д', 'Я', 'Ы', 'Х', 'Ь', 'К', 'Ч', 'Ф',
-#                'У', 'З', 'Ж', 'Г', 'Ю', 'Б', 'Й', 'Ш', 'Э', 'Ц', 'Ъ', 'Щ']
-#     """
-#     набор данных у которых точность некоторых символов одинаковая
-#     (что означает равновозможную вероятность появления некоторого одного символа вместо некоторого другого)
-#     """
-#
-#     equals_dict = dict()
-#     i = 0
-#     for etsf_key in sorted_text_dict.keys():
-#         j = 0
-#         for gsf_key in cor_key:
-#             j += 1
-#             equals_dict[etsf_key] = gsf_key
-#             if j > i: break
-#
-#         i += 1
-#     print(dict(zip(sorted_text_dict.keys(), cor_key)))
-#
-
-
-def task_2():
-    encrypted_text = read_file('task_2/cod8.txt')
-    ret_str = ''
-    # key = {'Я': ' ', 'Д': 'Е', ' ': 'А', '4': 'О', 't': 'И', 'М': 'Н', 'Р': 'Т', 'О': 'Р', 'П': 'С', 'Б': 'В', 'Л': 'М',
-    #        'К': 'Л', '5': 'П', '2': 'Д', 'Ь': 'Я', 'Щ': 'Ы', 'У': 'Х', 'Ъ': 'Ь', 'Й': 'К', 'r': 'З', 'Х': 'Ч', '>': 'У',
-    #        'Т': 'Ф', 'Е': 'Ж', '1': 'Г', 'Ы': 'Ю', 'А': 'Б', 'И': 'Й', 'Ц': 'Ш', '?': 'Ъ', 'Ф': 'Ц', 'Ч': 'Щ', 'w': 'Э'}
-
-    with open('task_2/key.json') as f:
-        key = json.load(f)
-
-    for symbol in encrypted_text:
-        try:
-            ret_str += key[symbol]
-        except KeyError:
-            ret_str += symbol
-    print(ret_str)
-
-
-def main() -> None:
-    # task_1('task_1/common_text.txt', 'task_1/cod8.txt')
-    task_2()
-
-
-if __name__ == '__main__':
-    main()
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
Index: lab_1/task_2/cod8.txt
===================================================================
diff --git a/lab_1/task_2/cod8.txt b/lab_1/task_2/cod8.txt
deleted file mode 100644
--- a/lab_1/task_2/cod8.txt	(revision 1b4bad32b7d100fe7ed0ea60f9fe53eac6cd1005)
+++ /dev/null	(revision 1b4bad32b7d100fe7ed0ea60f9fe53eac6cd1005)
@@ -1,1 +0,0 @@
-БЯОД КЪМ4ИЯЕtrМtЯЛМ41tДЯr 2 ХtЯМДЯЬБКЬЫРПЬЯПР4КЪЯ5ОtЛtРtБМЩЛtЯЙ ЙЯМ 5ОtЛДОЯr 2 Х Яr ЦtТО4Б МtЬЯ42М414ЯТ ИК Я4ХДМЪЯХ ПР4ЯРОДА>ДРПЬЯМДЯР4КЪЙ4ЯtЯМДПР4КЪЙ4Яr ЦtТО4Б РЪЯ2 ММЩДЯМ4ЯДЧДЯtЯП2ДК РЪЯtУЯУО МДМtДЯtЯ5ДОД2 Х>ЯЙ ЙЯЛ4ЕМ4ЯА4КДДЯwТТДЙРtБМДДЯБЯ5О41О ЛЛМЩУЯ5ОtК4ЕДМtЬУЯБ4БПЫЯtП54КЪr>ЫРПЬЯ К14ОtРЛЩЯПЕ РtЬЯ Я5ДОtТДОtИМЩДЯ>ПРО4ИПРБ ЯМ Я 55 О РМ4ЛЯ>О4БМДЯ5422ДОЕtБ ЫРЯО А4Р>ЯЛДР424БЯ4АДП5ДХДМtЬЯФДК4ПРМ4ПРtЯУО МtЛЩУЯtЯ5ДОД2 Б ДЛЩУЯ2 ММЩУ?
\ No newline at end of file
Index: lab_1/task_1/common_text.txt
===================================================================
diff --git a/lab_1/task_1/common_text.txt b/lab_1/task_1/common_text.txt
deleted file mode 100644
--- a/lab_1/task_1/common_text.txt	(revision 1b4bad32b7d100fe7ed0ea60f9fe53eac6cd1005)
+++ /dev/null	(revision 1b4bad32b7d100fe7ed0ea60f9fe53eac6cd1005)
@@ -1,3 +0,0 @@
-Лебеди – самые крупные представители водоплавающих птиц (вес лебедя-кликуна достигает 12 кг., размах крыльев – около двух метров). По сравнению с другими представителями семейства утиных, у лебедя наиболее длинная шея. Ее длина соответствует длине тела птицы. Плавает лебедь хорошо, однако как и всех водных птиц, его движения несколько медлительны. Но при опасности со стороны хищника лебедь способен плыть очень быстро. С водной поверхности птица взлетает не быстро – она долго разбегается, шлепая по воде лапами. Летают лебеди хорошо. Все представители лебедей являются перелетными птицами и способны преодолевать значительные расстояния.
-
-Оперенье лебедя имеет однородную окраску – белую, серую или черную. Половой диморфизм среди этих птиц практически не развит – самка и самец выглядят одинаково. Ноги у лебедя короткие. По земле птицы передвигаются не часто и походка у них вразвалку.
\ No newline at end of file
Index: lab_1/task_1/encrypted_text.txt
===================================================================
diff --git a/lab_1/task_1/encrypted_text.txt b/lab_1/task_1/encrypted_text.txt
deleted file mode 100644
--- a/lab_1/task_1/encrypted_text.txt	(revision 1b4bad32b7d100fe7ed0ea60f9fe53eac6cd1005)
+++ /dev/null	(revision 1b4bad32b7d100fe7ed0ea60f9fe53eac6cd1005)
@@ -1,3 +0,0 @@
-Ыхсхфш – брьлх ъагяэлх яахфбвртшвхыш тюфюяыртройше явшж (тхб ыхсхфп-ъышъгэр фюбвшурхв 12 ъу., арчьре ъалымхт – юъюыю фтге ьхвают). Яю бартэхэшо б фагушьш яахфбвртшвхыпьш бхьхщбвтр гвшэле, г ыхсхфп эршсюыхх фышээрп ихп. Хх фышэр бюювтхвбвтгхв фышэх вхыр явшжл. Яыртрхв ыхсхфм еюаюию, юфэръю ъръ ш тбхе тюфэле явшж, хую фтшцхэшп эхбъюымъю ьхфышвхымэл. Эю яаш юярбэюбвш бю бвюаюэл ешйэшър ыхсхфм бяюбюсхэ яылвм юзхэм слбваю. Б тюфэющ яютхаеэюбвш явшжр тчыхврхв эх слбваю – юэр фюыую арчсхурхвбп, иыхярп яю тюфх ырярьш. Ыхвров ыхсхфш еюаюию. Тбх яахфбвртшвхыш ыхсхфхщ птыповбп яхахыхвэльш явшжрьш ш бяюбюсэл яахюфюыхтрвм чэрзшвхымэлх арббвюпэшп.
-
-Юяхахэмх ыхсхфп шьххв юфэюаюфэго юъарбъг – схыго, бхаго шыш зхаэго. Яюыютющ фшьюадшчь бахфш нвше явшж яаръвшзхбъш эх арчтшв – брьър ш брьхж тлуыпфпв юфшэръютю. Эюуш г ыхсхфп ъюаювъшх. Яю чхьых явшжл яхахфтшуровбп эх зрбвю ш яюеюфър г эше тарчтрыъг.
\ No newline at end of file
Index: lab_1/task_2/frequency_symbols_based.txt
===================================================================
diff --git a/lab_1/task_2/frequency_symbols_based.txt b/lab_1/task_2/frequency_symbols_based.txt
deleted file mode 100644
--- a/lab_1/task_2/frequency_symbols_based.txt	(revision 1b4bad32b7d100fe7ed0ea60f9fe53eac6cd1005)
+++ /dev/null	(revision 1b4bad32b7d100fe7ed0ea60f9fe53eac6cd1005)
@@ -1,33 +0,0 @@
-§ = 0.128675
-О = 0.096456
-И = 0.075312
-Е = 0.072292
-А = 0.064841
-Н = 0.061820
-Т = 0.061619
-С = 0.051953
-Р = 0.040677
-В = 0.039267
-М = 0.029803
-Л = 0.029400
-Д = 0.026983
-Я = 0.026379
-К = 0.025977
-П = 0.024768
-З = 0.015908
-Ы = 0.015707
-Ь = 0.015103
-У = 0.013290
-Ч = 0.011679
-Ж = 0.010673
-Г = 0.009867
-Х = 0.008659
-Ф = 0.007249
-Й = 0.006847
-Ю = 0.006847
-Б = 0.006645
-Ц = 0.005034
-Ш = 0.004229
-Щ = 0.003625
-Э = 0.002416
-Ъ = 0.000000
\ No newline at end of file
diff --git a/lab_1/task_2/decipher_text.txt b/lab_1/task_2/decipher_text.txt
deleted file mode 100644
